tsconfig.json

```diff
@@ -17,5 +17,18 @@
     "strict": true,                                 /* Enable all strict type-checking options. */                      
     "skipLibCheck": true,                           /* Skip type checking of declaration files. */
     "forceConsistentCasingInFileNames": true        /* Disallow inconsistently-cased references to the same file. */
+  },
+  "typedocOptions": {
+    "name": "Autokit Helpers",
+    "entryPoints": ["lib/features/**", "lib/flashing/**", "lib/interfaces.d.ts", "lib/index.ts"],
+    "out": "./docs/",
+    "excludeExternals": true,
+    "excludePrivate": true,
+    "excludeProtected": true,
+    // "excludeNotDocumented": true,
+    "categorizeByGroup": true,
+    "categoryOrder": ["helper", "*"],
```

Comment:
You can probably remove this. Leviathan has this to explicitly mark methods that I want to show up as helper functions. This is not the case in autokit